### PR TITLE
Add reCAPTCHA to feedback form

### PR DESCRIPTION
**Pull request by michal-rohacek**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -60,6 +60,7 @@ gem 'sitemap_generator'
 
 # sendinblue V3
 gem 'sib-api-v3-sdk'
+gem "recaptcha"
```
Comment (pavolzbell) [inline Gemfile:63]:
```suggestion
gem 'recaptcha'
```

File: app/views/feedbacks/invalid_captcha.js.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+$('.feedback-submitted-feedback').text('Prosím, potvrďte, že nie ste robot.');
```
Comment (pavolzbell) [inline app/views/feedbacks/invalid_captcha.js.erb:1]: @HanaBene text je ok?

File: app/views/components/_feedbackbar.html.erb [hunk 3]
```diff
@@ -24,13 +21,32 @@
...
+          <a href="#" data-module="sdn-appear-link" data-appear="sdn-feedbackbar-container" data-disappear="sdn-feedbackbar-form-yes">Zatvoriť</a>
+        </div>
+      </div>
+      <%= form_with url: feedbacks_path do |form| %>
+        <%= hidden_field_tag :feedback_type, 'Useful' %>
+        <%= hidden_field_tag :current_path, request.fullpath %>
+        <div class="govuk-form-group">
+          <%= recaptcha_tags hl: 'sk' %>
+        </div>
+        <%= form.submit 'Odoslať', class: 'govuk-button', data: { "disable-with": "Odosielam..." } %>
```
Comment (pavolzbell) [inline app/views/components/_feedbackbar.html.erb:39]:
```suggestion
        <%= form.submit 'Odoslať', class: 'govuk-button', data: { 'disable-with': 'Odosielam...' } %>
```

File: app/views/feedbacks/invalid_captcha.js.erb (hunk 2 above)
Comment (HanaBene) [reply app/views/feedbacks/invalid_captcha.js.erb:1]: No objections here.

**Pull request merged**